src/components/client/support-form/ValidationObserver.tsx

```diff
@@ -0,0 +1,39 @@
+import { useFormikContext } from 'formik'
+import { useEffect, useRef } from 'react'
+import { Steps } from './helpers/support-form.types'
+
+const STEPS_VALIDATION_MAP = {
+  [Steps.ROLES]: ['roles'],
+  [Steps.QUESTIONS]: ['benefactor', 'partner', 'volunteer', 'associationMember', 'company'],
+  [Steps.PERSON]: ['person'],
+}
+
+const ValidationObserver = ({
```

Comment:
Let's add some comment stating the need to turn off individual field validation first time the step is submitted but then turning it back on so the user gets immediate feedback when filling the correct/expected value

Reply:
And finally - this new pattern might benefit greatly from a few unit tests. Benefit in terms of readability.